src/shell/CMakeLists.txt

```diff
@@ -7,10 +7,16 @@ add_library(Shell STATIC
     Prompt.cpp Prompt.h
     Shell.cpp Shell.h
     TTY.cpp TTY.h
-    UnixPipe.cpp UnixPipe.h
     Visitor.h
 )
 
+
+target_sources(Shell
+  PUBLIC
+    FILE_SET CXX_MODULES FILES
+      UnixPipe.cpp
```

Comment:
shouldn't the other files, such as Shell.cpp/h, TTY.cpp/h also be part of the module set?